README.md

```diff
@@ -10,13 +10,13 @@ Please note that this extension is currrently in **alpha stage** and under **act
 * CiviCRM `v5.8`+, `v5.12`+ recommended
 * [uk.squiffle.kam](https://github.com/aydun/uk.squiffle.kam) if the CiviCRM version is lower than `v5.12`
 * "CiviCRM Theme" module enabled
-* "Seven" theme set as CiviCRM Administration theme
+* The [CiviAdmin Companion Drupal theme](https://github.com/compucorp/shoreditch-companion-d7-theme) is optional but  recommended
 
 ### For development
 * NodeJS `v10.15.3`
 
 ## Supported CMSs
-At the moment the theme is being developed to work only in Drupal 7 (with default theme set to "Seven"). WordPress and Joomla are not currently supported.
+At the moment the theme is being developed to work only in Drupal 7. WordPress and Joomla are not currently supported.
```

Comment:
Is this text necessary anymore? Because after decoupling, shoreditch should not care about what CMS is supports.